app/locale.js

```diff
@@ -255,7 +255,10 @@ var rendererIdentifiers = function () {
     'connectionError',
     'unknownError',
     'allowAutoplay',
-    'autoplayMedia'
+    'autoplayMedia',
+    // Release channels
+    'channelDev',
```

Comment:
New translatable strings must be entered here in order to be accessible by the `locale.translate()` method.  The name here matches the name in `en-US/app.properties`